python/lambdas/copy-files-from-tdr/copy_files_lambda_function_test.py

```diff
@@ -156,7 +156,8 @@ def test_send_message_failure(self, mock_validate_metadata, mock_send_message, _
 
     def test_should_successfully_validate_when_the_fields_are_valid(self):
         mock_response_body = json.dumps(self.valid_metadata())
-        result = lambda_function.validate_mandatory_fields_exist(json.loads(mock_response_body))
+        schema_location = "python/lambdas/copy-files-from-tdr/metadata-schema.json"
```

Comment:
Should this be a global variable?

Reply:
I'm going to leave it. Too many global variables in tests makes them hard to read as you're jumping around everywhere to see what it's doing. These tests aren't long so it's fine.